pxr/imaging/plugin/hdEmbree/renderer.cpp

```diff
@@ -960,38 +1458,63 @@ HdEmbreeRenderer::_ComputeColor(RTCRayHit const& rayHit,
     // Make sure the normal is unit-length.
     normal.Normalize();
 
-    // Lighting model: (camera dot normal), i.e. diffuse-only point light
-    // centered on the camera.
-    GfVec3f dir = GfVec3f(rayHit.ray.dir_x, rayHit.ray.dir_y, rayHit.ray.dir_z);
-    float diffuseLight = fabs(GfDot(-dir, normal)) *
-        HdEmbreeConfig::GetInstance().cameraLightIntensity;
+    GfVec3f lightingColor(0.0f);
+
+    // If there are no lights, then keep the existing camera light + AO path to
+    // be able to inspect the scene
+    if (_lightMap.empty())
```

Comment:
The main change I've made locally is to add a render setting "enableLighting" and a corresponding env setting HDEMBREE_USE_LIGHTING to turn on the light computation, as opposed to AO.  Happy to rename those if you'd like.  I also set that setting to false by default, though it should be easy to enable in usdview and then it'll get stored in user state forever.  If "enableLighting" is on, it overrides AO settings; if it's off, we fall back to AO.

Having it as an explicit mode seems preferable, and also makes it easier to merge what with the unit tests/etc.  But let me know your thoughts on this.